lib/web_ui/lib/src/engine/html/color_filter.dart

```diff
@@ -60,15 +60,20 @@ class PersistedColorFilter extends PersistedContainerSurface
       childContainer?.style.visibility = 'visible';
       return;
     }
-    if (engineValue._blendMode == null) {
-      rootElement!.style.backgroundColor =
-          colorToCssString(engineValue._color!);
+    // if (engineValue._blendMode == null) {
+    //   rootElement!.style.backgroundColor =
+    //       colorToCssString(engineValue._color!);
```

Comment:
Not sure I understand this part. If `_blendMode` is null then isn't `_color` supposed to be null too? @ferhatb

Reply:
Defensive code. If there is an assert at color filter level, we can drop the second if.